index.html

```diff
@@ -164,9 +162,8 @@ <h2>Tonic</h2>
             <li>html</li>
             <li>css</li>
             <li>javascript</li>
-            <li>Ruby on rails</li>
           </ul>
-          <a href="#" class="btn btn-animation">See Project</a>
+          <button href="#" class="btn btn-animation" id="project-four">See Project</button>
```

Comment:
- [ ] Since you are asked to create the project list dynamically from your js file. All the already created code for the project list in your index.html file should be removed. Kindly remove all the already created html code for the project list.